components/Chrome.tsx

```diff
@@ -43,8 +38,19 @@ const Chrome: React.FC<Props> = ({ page, smallFooter, children }) => {
     );
   }
 
+  function toggleLanguage(): void {
+    if (language === 'en') {
+      setLang('es');
+      setCookie('languageOverride', 'es');
+    } else {
+      setLang('en');
+      setCookie('languageOverride', 'en');
+    }
+    router.push(redirectTo);
```

Comment:
what happens on the homepage? do we want to _redirect_ to the same page? does it cause a refresh?

Reply:
tested it briefly, doesn't seem to cause a refresh, but i still wonder if it would be better to not push if the redirect was `/`. i think that theoretically dorks up clicking the "back" button.